contracts/cw20-merkle-airdrop/Cargo.toml

```diff
@@ -22,17 +22,17 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cw-utils = { version = "0.11.1" }
-cw2 = { version = "0.11.1" }
-cw20 = { version = "0.11.1" }
-cosmwasm-std = { version = "1.0.0-beta3" }
-cw-storage-plus = { version = "0.11.1" }
+cw-utils = { version = "0.11" }
+cw2 = { version = "0.11" }
+cw20 = { version = "0.11" }
+cosmwasm-std = { version = "1.0.0-beta" }
+cw-storage-plus = { version = "0.11" }
 schemars = "0.8.1"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 thiserror = { version = "1.0.23" }
 hex = "0.4"
 sha2 = { version = "0.9.5", default-features = false }
 
 [dev-dependencies]
-cosmwasm-schema = "1.0.0-beta3"
+cosmwasm-schema = "1.0.0-beta"
```

Comment:
Why downgrade?

Reply:
This is not a downgrade, rather a cool cargo trick. This works the same way as a minor change, `1.0.0-betaX` cargo will follow the latest minor version.